Utilities/StrFmt.cpp

```diff
@@ -660,6 +660,17 @@ bool fmt::match(const std::string& source, const std::string& mask)
 	return true;
 }
 
+std::string fmt::aligned_padding(const std::string& input, usz length, bool right_align, char padding_char)
```

Comment:
fmt::format("%8s", input) should already do that iirc, this pads it to 8 characters.

Reply:
How about padding from left?

Reply:
It's irrelevant since the format string needs to be hardcoded iirc.

Reply:
Well, hardcoded is not completely true. fmt::format(".*s", pad_count, string);
The * is for dynamic length.
What is hardcoded are the padding chars (0 or spaces) and the direction.

Reply:
I was also testing something.
It turns out that `printf("%**s", length * right_align ? 1 : -1, input)` can work, but neither the former C style format nor your suggestion works with RPCS3's fmt::format().
Output of `fmt::format(".*s", arg->serial_size - 1, g_cfg.sys.hdd_serial.to_string())`: `".*s"`